reference/configuration/framework.rst

```diff
@@ -1446,6 +1462,134 @@ value.
     For more information, see
     :ref:`component-serializer-converting-property-names-when-serializing-and-deserializing`.
 
+.. _reference-cache:
+
+cache
+~~~~~
+
+.. _reference-cache-app:
+
+app
+...
+
+**type**: ``string`` **default**: ``cache.adapter.filesystem``
+
+The cache adapter used by the ``cache.app`` service.
+
+.. tip::
+
+    It might be tough to understand at the beginning, so to avoid confusion remember that all pools perform the
+    same actions but on different medium given the adapter they are based on. Internally, a pool wraps the definition
+    of an adapter.
+
+.. note::
+
+    The framework bundle ships with multiple adapters: apcu, doctrine, system,
+    filesystem, psr6, and redis.
+
+system
+......
+
+**type**: ``string`` **default**: ``cache.adapter.system``
+
+The cache adapter used by the ``cache.system`` service.
+
+directory
+.........
+
+**type**: ``string`` **default**: ``%kernel.cache_dir%/pools``
+
+The path to the cache directory used by services inheriting from the
+``cache.adapter.filesystem`` adapter (including ``cache.app``).
+
+default_doctrine_provider
+.........................
+
+**type**: ``string``
+
+The service name to use as your default Doctrine provider.
```

Comment:
Maybe add the service name of this "default provider": "This is available as ``cache.doctrine`` service."